src/providers/oracle/ocispatial/qsql_ocispatial.cpp

```diff
@@ -1181,6 +1195,23 @@ class QOCISpatialCols
 
     QSqlRecord rec;
 
+    struct Point
+    {
+      Point( double x = 0, double y = 0, double z = 0 )
+        : x( x )
+        , y( y )
+        , z( z )
```

Comment:
No `m` value?
Is there no structure we can use for this?

Reply:
because it's LGPLed and not using GPLed code - to be able to link it to the OCI library.

Reply:
Ah, that makes sense. I thought it was purely to keep the code seperate in case anyone else wanted to use the driver.

Reply:
Oh I see. Ignore my comments.